todo_list_app/serializers.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+from todo_list_app.models import User
+
+
+class UserRegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(max_length=68,min_length=6,write_only=True)
+    password2 = serializers.CharField(max_length=68,min_length=6,write_only=True)
+
+    class Meta:
+        model = User
+        fields = ['email', 'first_name', 'last_name', 'password', 'password2']
+
+    def validate(self, attrs):
+        password = attrs.get('password','')
+        password2 = attrs.get('password2','')
+        if password != password2:
+            raise serializers.ValidationError('Passwords must match')
+        return attrs
+
+    def create(self, validated_data):
```

Comment:
When will this method be called?

Reply:
First of all an instance of serializer will be created after this on serialize.is_valid it runs validate function
after this on serialzier.save it runs create function if on creation serializer instance you passed only one parameter "data"
but if you passed two parameters like "instance" and "data" then it will call update function of serializer